### PR TITLE
ToDo app

### PR DESCRIPTION
**Pull request by leelalaim**
Link to my deployed project: https://tasks-to-do-app.netlify.app/

### CODE REVIEW

**Review by jessika-hage — APPROVED** [review]
I don't have anything to comment on other than that it looks great and the code is very neat and easy to understand! Great job! :D

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+}
+
+const store = createStore(
+  reducer,
+  persistedState,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+);
+
+store.subscribe(() => {
+  localStorage.setItem("reduxState", JSON.stringify(store.getState()));
```
Comment (KarolCislowski) [inline code/src/App.js:30]: Persistence 😻

File: code/src/components/TaskInput.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  return (
+    <>
+      <form onSubmit={onFormSubmit} className="input-wrapper">
+        <input
+          placeholder="Add task..."
+          className="task-input"
+          type="text"
+          value={task}
+          maxLength="50"
+          onChange={(e) => setTask(e.target.value)}
```
Comment (KarolCislowski) [inline code/src/components/TaskInput.js:34]: FIGHT WITH LAZYNESS!! 🔫  add REQUIRED to make your to-do free from empty tasks!! 🎖️  😆

File: code/src/components/TaskInput.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      isComplete: false,
+    };
+
+    dispatch(todos.actions.addTask(newTask));
+
+    setTask("");
+  };
+
+  return (
+    <>
```
Comment (KarolCislowski) [inline code/src/components/TaskInput.js:26]: You have only 1 element (`<form>`) to return here so you shouldn't use <></>

**Review by KarolCislowski — APPROVED** [review]
Great app. For the next step, it would be nice to make it look better on the desktop screen 💻

**Pull request closed without merging**